spec/services/list_languages_taught_in_modern_language_course_spec.rb

```diff
@@ -0,0 +1,59 @@
+describe ListLanguagesTaughtInModernLanguageCourse do
+  describe '#call' do
+    context "when the course isn't for modern languages" do
+      let(:course_name) { 'Physical Education with French' }
+
+      context 'with two language subjects' do
+        let(:ucas_subject_names) do
+          'French, German, Languages, Languages (European), Secondary'
+        end
+
+        subject { described_class.call(course_name, ucas_subject_names) }
+
+        it { should eq([]) }
+      end
+    end
+
+    context "when the course name contains MFL" do
+      let(:course_name) { 'MFL - Spanish' }
+
+      context 'with two language subjects' do
+        let(:ucas_subject_names) do
```

Comment:
any items that this touches 
https://github.com/DFE-Digital/manage-courses-backend/blob/590a40c71e630d979d7938464a2a639e7e26b6f7/app/services/subject_mapper.rb#L106

we set the get_subjects_list should return either `"Modern languages" or` "Modern languages (other)"